firmware/controllers/actuators/dc_motors.cpp

```diff
@@ -11,76 +11,76 @@
 
 #include "dc_motors.h"
 
-	void DcHardware::start(bool useTwoWires,
-			brain_pin_e pinEnable,
-			brain_pin_e pinDir1,
-			brain_pin_e pinDir2,
-			brain_pin_e pinDisable,
-			bool isInverted,
-			ExecutorInterface* executor,
-			int frequency) {
-
-		if (isStarted) {
-			// actually implement stop()
-			return;
-		}
-		isStarted = true;
-
-		dcMotor.setType(useTwoWires ? TwoPinDcMotor::ControlType::PwmDirectionPins : TwoPinDcMotor::ControlType::PwmEnablePin);
-
-		// Configure the disable pin first - ensure things are in a safe state
-		m_disablePin.initPin("ETB Disable", pinDisable);
-		m_disablePin.setValue(0);
-
-		// Clamp to >100hz
-		int clampedFrequency = maxI(100, frequency);
-
-		if (clampedFrequency > ETB_HW_MAX_FREQUENCY) {
-			firmwareError(ObdCode::OBD_PCM_Processor_Fault, "Electronic throttle frequency too high, maximum %d hz", ETB_HW_MAX_FREQUENCY);
-			return;
-		}
-
-		if (useTwoWires) {
-			m_pinEnable.initPin("ETB Enable", pinEnable);
-
-// no need to complicate event queue with ETB PWM in unit tests
-#if ! EFI_UNIT_TEST
-			startSimplePwmHard(&m_pwm1, "ETB Dir 1",
-				executor,
-				pinDir1,
-				&m_pinDir1,
-				clampedFrequency,
-				0
-			);
-
-			startSimplePwmHard(&m_pwm2, "ETB Dir 2",
-				executor,
-				pinDir2,
-				&m_pinDir2,
-				clampedFrequency,
-				0
-			);
-#endif // EFI_UNIT_TEST
-
-			dcMotor.configure(wrappedEnable, m_pwm1, m_pwm2, isInverted);
-		} else {
-			m_pinDir1.initPin("ETB Dir 1", pinDir1);
-			m_pinDir2.initPin("ETB Dir 2", pinDir2);
-
-// no need to complicate event queue with ETB PWM in unit tests
-#if ! EFI_UNIT_TEST
-			startSimplePwmHard(&m_pwm1, "ETB Enable",
-				executor,
-				pinEnable,
-				&m_pinEnable,
-				clampedFrequency,
-				0
-			);
-#endif // EFI_UNIT_TEST
-
-			dcMotor.configure(m_pwm1, wrappedDir1, wrappedDir2, isInverted);
-		}
+void DcHardware::start(bool useTwoWires,
```

Comment:
this is just an indentation cleanup

Reply:
unrelated changes in an unrelated PR, sigh

Reply:
I'll drop the last two commits and put them in separate PRs, no worries.